README.md

```diff
@@ -52,39 +52,40 @@ Each API Service contains its own integration test suite that tests it against t
 ### Coordinator Node
```

Comment:
Something is wrong right above this. These lines don't belong:

```
**Warning:** Support for Cassandra 3.11 is considered deprecated and will be removed in the Stargate v3 release: [details](https://github.com/stargate/stargate/discussions/2242).
**Warning:** Support for Cassandra 3.11 is considered deprecated and will be removed in the Stargate v3 release: [details](https://github.com/stargate/stargate/discussions/2242).
- **Authentication Services**: Responsible for authentication to Stargate
```

Reply:
I did not add this. Will remove.